npm-shrinkwrap.json

```diff
@@ -4895,7 +4900,7 @@
     "yoastseo": {
       "version": "1.2.2",
       "from": "git+https://github.com/artsy/YoastSEO.js.git#develop",
-      "resolved": "git+https://github.com/artsy/YoastSEO.js.git#98ed2d2adf15bb9ef5ae95a01269bb7f21c24638"
+      "resolved": "git+https://github.com/artsy/YoastSEO.js.git#a1bbf40ec2dbc807412ca57374a65d95e37c0fcd"
```

Comment:
Hmm looks like this got resolved back to https://github.com/artsy/YoastSEO.js/commit/a1bbf40ec2dbc807412ca57374a65d95e37c0fcd

Could we move to the latest `638`?